hpilo_fw.py

```diff
@@ -62,10 +62,10 @@ def download(ilo, path=None, progress = lambda txt: None):
     return False
 
 def parse(fwfile, ilo):
-    fd = open(fwfile)
+    fd = open(fwfile, 'rb')
     data = fd.read()
     fd.close()
-    if '_SKIP=' in data:
+    if b'_SKIP=' in data:
```

Comment:
This change unfortunately needs to be done in a different way (no b'' strings), as I still support python 2.4 for the poor sods stuck on rhel 5.